cloud/disk_manager/internal/pkg/dataplane/nbs/source.go

```diff
@@ -125,6 +125,7 @@ func (s *diskSource) generateChunkIndices(
 			chunkIndex++
 		}
 
+		s.chunkIndices.UpdateDefaultMilestoneValue(chunkIndex)
```

Comment:
If the source calls the UpdateDefaultMilestoneValue method with some value v, it promises that it will not send values less then v to the queue anymore. Maybe I should write comment about it.

Reply:
да, предлагаю еще подумать над названием. и комммент точно нужен

Reply:
в целом может быть имеет смысл обновлять milestone только в этом месте?
это облегчит логику, но появятся повторные копирования (если дм рестартанет до того, как обновит milestone, например)

Reply:
Про название/комментарий согласен -- надо бы ещё раз это продумать.

А насчёт обновлять milestone только в этом месте -- возможно я не до конца понял, что ты имеешь в виду. Но обновлять milestone только в disk source не получится, т.к. source не должен знать про то, когда тот или иной чанк закончит обрабатываться. Как я понимаю, это зона ответственности inflight queue.